client/extensions/woocommerce/app/store-stats/store-stats-navigation/style.scss

```diff
@@ -0,0 +1,40 @@
+.followers-count {
+	.button {
+		margin-right: 16px;
+	}
+}
+
+&.segmented-control,
+.segmented-control {
+	margin-right: 16px;
+
+	.segmented-control__item {
+
+		.segmented-control__link {
+			background: #f3f6f8;
+			border-color: #e9eff3;
+
+			.segmented-control__text {
+				color: #a8bece;
+			}
+		}
+
+		&.is-selected {
+			.segmented-control__link {
+				background: #00aadc;
+				border-color: #008ab3;
+
+				.segmented-control__text {
+					color: #ffffff;
+				}
+			}
+		}
+
+		&.is-selected + .segmented-control__item {
+			.segmented-control__link {
+				border-left-color: #008ab3;
+			}
+		}
+
+	}
```

Comment:
A little something I noticed on mobile: The pill has a space to its right.

![screen shot 2017-06-14 at 12 05 49 pm](https://user-images.githubusercontent.com/1922453/27110479-c8876cfc-50fd-11e7-8af9-e8cac9e2da72.png)